task_02/style.css

```diff
@@ -0,0 +1,281 @@
+* {
+  font-family: "Google Sans", Roboto, arial, sans-serif;
+}
+
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.mobile-navigation {
+  display: none;
+}
+
+.wrapper {
+  display: flex;
+}
+
+nav {
+  flex: 0 0 22vw;
+  background: #384043;
+}
+
+.nav-wrapper {
+  position: fixed;
+  top: 0;
+  left: 0;
+}
+
+a#disabled {
+  color: #83878a;
+}
+a#disabled li {
+  color: #83878a;
+}
+
+section {
+  flex: 1 1;
+  width: 100%;
+}
+
+.item {
+  height: 80vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+#item1 {
+  background-color: #68d2e7;
+}
+
+#item2 {
+  background-color: #f9d423;
+}
+
+#item3 {
+  background-color: #f3862f;
+}
+
+#item4 {
+  background-color: #ff4e4f;
+}
+
+.section-wrapper {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.text-wrapper {
+  margin-left: 10%;
+  margin-right: 10%;
+}
+
+.list-item {
+  margin-left: 40px;
+  margin-top: 20px;
+}
+
+.links {
+  text-decoration: none;
+  font-size: 20px;
+  color: #fff;
+  font-weight: bolder;
+}
+
+.post-title {
+  color: #fff;
+  font-size: 28px;
+  font-weight: 900;
+}
+
+.post-text {
+  color: #fff;
+  font-size: 14px;
+  font-weight: 200;
+  word-break: break-all;
+}
+
+@media (min-width: 576px) and (max-width: 768px) {
+  nav.desktop-navigation {
+    flex: 0 0 30vw;
+  }
+}
+
+@media (max-width: 576px) {
+  .desktop-navigation {
+    display: none;
+  }
+
+  .mobile-navigation {
+    position: fixed;
+    display: block;
+    background-color: transparent;
+    z-index: 1;
+  }
+
+  section {
+    z-index: 0;
+  }
+
+  /* Mobile styling */
+  body {
+    margin: 0;
+    padding: 0;
```

Comment:
Well, for padding it isn't, since its default value is 0. But on the other hand, default value in most major browsers for margin is 8px so if we want to display it without margin, it is necessary.